src/blocks/Expenses/ExpenseCard/ExpenseCard.jsx

```diff
@@ -0,0 +1,21 @@
+import Icon from "../../../components/Icon";
+import "./ExpenseCard.css";
+
+const ExpenseCard = (props) => {
+  return (
+    <div className="expense">
+      {props.icon ? (
```

Comment:
can we change this conditional to `{props.icon && (...)}`. I mean, `&& ...` instead of ` ? ... : ...`